src/configs/ubiquibot-config-default.ts

```diff
@@ -1,18 +1,18 @@
 import { MergedConfig } from "../types";
 
 export const DefaultConfig : MergedConfig = {
-  "evm-network-id": 100,
-  "price-multiplier": 1,
-  "issue-creator-multiplier": 2,
-  "payment-permit-max-price": 9007199254740991,
-  "max-concurrent-assigns": 9007199254740991,
-  "assistive-pricing": false,
-  "disable-analytics": false,
-  "comment-incentives": false,
-  "register-wallet-with-verification": false,
-  "promotion-comment": "\n<h6>If you enjoy the DevPool experience, please follow <a href='https://github.com/ubiquity'>Ubiquity on GitHub</a> and star <a href='https://github.com/ubiquity/devpool-directory'>this repo</a> to show your support. It helps a lot!</h6>",
-  "default-labels": [],
-  "time-labels": [
+  "evmNetworkId": 100,
+  "priceMultiplier": 1,
+  "issueCreatorMultiplier": 2,
+  "paymentPermitMaxPrice": 9007199254740991,
+  "maxConcurrentAssigns": 9007199254740991,
+  "assistivePricing": false,
```

Comment:
you don't need quotes (`"`) when using camelCase
```suggestion
  assistivePricing: false,
```
update this everywhere